src/Tribe/RSVP.php

```diff
@@ -1225,14 +1225,14 @@ public function get_ticket( $event_id, $ticket_id ) {
 		$start_date_unix          = strtotime( $start_date );
 		if ( ! empty( $start_date ) ) {
 			$return->start_date = date( 'Y-m-d', $start_date_unix );
-			$return->start_time = date( 'h:ia', $start_date_unix );
+			$return->start_time = Tribe__Date_Utils::time_only( $start_date_unix );
 		}
 
 		$end_date                 = get_post_meta( $ticket_id, '_ticket_end_date', true );
 		$end_date_unix            = strtotime( $end_date );
```

Comment:
You can move this into the `if ( ! empty( $end_date ) )` since it's not used outside of it - and there's no reason to generate it if `$end_date` is empty?